StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1130UseLambdaSyntax.cs

```diff
@@ -96,6 +96,28 @@ internal static ParameterListSyntax GetDelegateParameterList(ISymbol symbol, int
             return SyntaxFactory.ParameterList(SyntaxFactory.SeparatedList(syntaxParameters));
         }
 
+        internal static int FindArgumentIndex(SymbolInfo originalSymbolInfo, ArgumentSyntax argumentSyntax, BaseArgumentListSyntax argumentListSyntax)
+        {
+            // if delegate is passed as named argument of method try to find its position by argument name
+            if (originalSymbolInfo.Symbol.Kind == SymbolKind.Method
```

Comment:
❔ Is this condition required since we have the `IMethodSymbol` check?

Reply:
Not required really, my intention was to compare enum value quickly before doing typecast (`is`), I will remove that condition.